corehq/apps/app_manager/tests/data/suite/suite-advanced-commtrack.xml

```diff
@@ -294,9 +294,26 @@
         <locale id="forms.m0f0"/>
       </text>
     </command>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <instance id="commcaresession" src="jr://instance/session"/>
```

Comment:
I had to add in these nodes for the commtrack suite, as this is default behavior for all apps on a commtrack domain.